pg_metadata/metadata.txt

```diff
@@ -6,7 +6,7 @@ author=3Liz
 email=info@3liz.com
 about=Store some metadata about layers which are stored in PostgreSQL. It's possible to define title, abstract, links, contacts etc. It also provides some search tool, in the QGIS Locator bar for instance.
 
-version=master
+version=1.2.0
```

Comment:
Same comment, you would need to revert this change. It's done automatically by the CI with https://github.com/opengisch/qgis-plugin-ci

Reply:
Thanks for the info!